torchvision/datasets/video_utils.py

```diff
@@ -0,0 +1,77 @@
+import bisect
+import torch
+from torchvision.io import read_video_timestamps, read_video
+
+
+def unfold(tensor, size, step, dilation):
```

Comment:
this uses stride tricks to compute all the possible clips in the video, with potential steps between clips and dilation (steps between frames)